Automattic/simplenote-ios
# Restore default link data detector

About 4 years ago, we rolled our own linkifier in order to address a performance issue when loading a note with a large amount of links in it.

I've tested the default linkifier here in _the future_ and have not observed any noticeable slowdowns when loading a note that contains hundreds of links, even on an older iPad mini. So I think we're safe to restore the default behavior.

**To Test**
* Create a whopper of a note that contains lots of text and links. I used this, pasted a few times over: https://moz.com/top500
* Load the note in the editor, you should see the links become blue.
* Tapping inside of the note text will disable the links when the soft keyboard shows.

Fixes #246
